package.json

```diff
@@ -164,7 +164,7 @@
     "copy-webpack-plugin": "10.2.4",
     "cross-env": "7.0.3",
     "css-loader": "6.7.1",
-    "electron": "29.0.0",
+    "electron": "29.1.0",
```

Comment:
Should this be updated here or separately when all dependencies are scheduled for upgrade?

Reply:
I was seeing few tests break due to white screen. I am check i case the upgrade would fix it. I'll revert this.